src/models/sheltersModel.js

```diff
@@ -0,0 +1,18 @@
+const db = require("./db.js");
+
+const getAllShelters = async (params) => {
+    const sql = `SELECT * FROM Shelter`;
+    return new Promise((resolve, reject) => {
```

Comment:
Great job figuring this out again! I had to do some research because I thought returning a Promise in an async function was a code smell, but since it is actually for the mysql query, it seems like it has been deemed okay.

Example:
https://stackoverflow.com/questions/48647340/node-mysql-connects-but-returns-unreadable-output